packages/sanity/src/core/form/inputs/PortableText/hooks/useScrollSelectionIntoView.tsx

```diff
@@ -16,7 +16,7 @@ export function useScrollSelectionIntoView(scrollElement: HTMLElement | null) {
           scrollIntoView(leafEl, {
             scrollMode: 'if-needed',
             boundary: scrollElement,
-            block: 'start',
+            block: 'nearest',
```

Comment:
This definitely improves the experience 👍  However, the editor still jumps, and I think it would be great if we could figure out how to prevent it from jumping at all when deleting content.

Reply:
This happens because we have padding at the bottom of the text. I think we would have to implement custom scrolling logic for this and check if we are at the bottom and if the `leafEl` is at the bottom of the viewport and then _not_ scroll if this is the case. I do not think this would be worth it, but if you have another suggestion, then I am open to it!